RNA-seq/07-bulk_rnaseq_exercise.Rmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "Exercise: Leukemia bulk RNA-seq"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+Here, we'll examine a mouse model of acute myeloid leukemia (AML).
+We're using [`SRP049821`](https://www.ebi.ac.uk/ena/data/view/PRJNA267255).
+Here's the description from the authors:
+
+> Mutant RAS oncoproteins activate signaling molecules that drive oncogenesis in multiple human tumors including acute myelogenous leukemia (AML). However, the specific function of these pathways in AML is unclear. To elucidate the downstream functions of activated NRAS in AML, we employed a murine model of AML harboring Mll-AF9 and NRASG12V. We found that NRASG12V enforced leukemia self-renewal gene expression signatures and was required to maintain an MLL-AF9 and MYB-dependent gene expression program. In a multiplexed analysis of RAS-dependent signaling intermediates, the leukemia stem cell compartment was preferentially sensitive to RAS withdrawal. Use of RAS-pathway inhibitors showed that NRASG12V maintained leukemia self-renewal through mTOR and MEK pathway activation, implicating these pathways as potential targets for cancer stem cell-specific therapies. Overall design: Primary leukemia cells harvested from spleens were sorted into immunophenotypic subpopulations (Mac-1High, Mac-1LowKit–Sca-1–, Mac-1LowKit+Sca-1–, and Mac-1LowKit+Sca-1+). RNA was extracted from this subpopulations of cells and submitted for RNA sequencing.
+
+These data were published as part of Sachs et al. _Blood._ 2014.
+
+### Setup 
+
+We've prepared the expression data all the way through the tximport step.
+This file can be found at: `data/tximport/leukemia/leukemia_stem_cell_txi.RDS`
+
+The metadata for this experiment can be found in this TSV file: `data/sample_metadata/leukemia/SRP049821_metadata.tsv`
+
+Using what we've learned so far, let's ask a series of questions about these
+data using DESeq2.
+
+```{r}
+library(DESeq2)
+```
+
+We'll use the `%>%` operator as well.
+
```

Comment:
`We've created an alias for you` ?

Reply:
That seems too technical to me. I think this statement is more consistent with how we talk about this throughout the course.